Content.Server/Medical/DefibrillatorSystem.cs

```diff
@@ -116,7 +118,15 @@ public bool CanZap(EntityUid uid, EntityUid target, EntityUid? user = null, Defi
         if (TryComp(target, out CMDefibrillatorBlockedComponent? block))
         {
             if (user != null)
-                _popup.PopupEntity(Loc.GetString(block.Popup), uid, user.Value);
+                _popup.PopupEntity(Loc.GetString(block.Popup, ("target", target)), uid, user.Value);
+            return false;
+        }
+
+        if (_inventory.TryGetSlotEntity(target, "outerClothing", out var armor)
+            && TryComp(armor, out CMDefibrillatorBlockedComponent? comp))
```

Comment:
This check should be done by using _inventory.GetSlotEnumerator(target, SlotFlags.OUTERCLOTHING) instead and checking if any of the items in those slots have the component